Lib/ctypes/test/test_bitfields.py

```diff
@@ -233,6 +233,78 @@ class X(Structure):
         else:
             self.assertEqual(sizeof(X), sizeof(c_int) * 2)
 
+    @unittest.skipIf(os.name == 'nt', reason='Posix only')
```

Comment:
What is the reason this behavior only affects Posix? Could you include that in the reason? Something like `reason='Only Posix has packed bitfields'`?

Reply:
Hi, Windows also has packed bitfields, though there is some special handling to it. As I don't have a Windows machine, and the reported issue was specifically for the Posix code path, I did not touch the Windows implementation.

Reply:
Fair enough.